api/spec/controllers/mno_enterprise/jpi/v1/admin/organizations_controller_spec.rb

```diff
@@ -43,11 +43,13 @@ def partial_hash_for_arrears(arrear)
     let(:org_invite) { build(:org_invite, organization: organization) }
     let(:app_instance) { build(:app_instance, organization: organization) }
     let(:credit_card) { build(:credit_card, organization: organization) }
+    let(:app) { build(:app)}
 
     before do
       organizations = [organization]
       allow(organizations).to receive(:loaded?).and_return(true)
       allow_any_instance_of(MnoEnterprise::User).to receive(:organizations).and_return(organizations)
+      allow_any_instance_of(MnoEnterprise::AppInstance).to receive(:app).and_return(app)
```

Comment:
Doesn't that mean we have a N+1 here?
* 1 Org with N App Instances
* Fetch N apps?